### PR TITLE
Add repos filter to restrict the repos we check 

### PR DESCRIPTION
**Pull request by markddavidoff**

### CODE REVIEW

File: slack_pull_reminder.py [hunk 1]
```diff
@@ -33,7 +37,7 @@ def fetch_repository_pulls(repository):
 def is_valid_title(title):
     lowercase_title = title.lower()
     for ignored_word in IGNORE_WORDS:
-        if ignored_word.lower() in lowercase_title:
+        if ignored_word in lowercase_title:
```
Comment (ekmartin) [inline slack_pull_reminder.py:40]: Why did you remove `.lower()` here?

File: slack_pull_reminder.py [hunk 2]
```diff
@@ -7,7 +7,11 @@
 POST_URL = 'https://slack.com/api/chat.postMessage'
 
 ignore = os.environ.get('IGNORE_WORDS')
-IGNORE_WORDS = ignore.split(',') if ignore else []
+IGNORE_WORDS = [i.lower().strip() for i in ignore.split(',')] if ignore else []
+
+repos = os.environ.get('REPOS')
```
Comment (ekmartin) [inline slack_pull_reminder.py]: Let's call it `repositories` and `REPOSITORIES`

**Comment by markddavidoff** [comment]
It's moved to the generator at the top

File: slack_pull_reminder.py [hunk 3]
```diff
@@ -61,6 +65,9 @@ def fetch_organization_pulls(organization_name):
     lines = []
 
     for repository in organization.repositories():
+        if REPOS and repository.name.lower() not in REPOS:
+            # if repos list is defined, skip repos not defined in filter
```
Comment (ekmartin) [inline slack_pull_reminder.py]: Think you can remove this comment, the `if` speaks for itself :)

File: slack_pull_reminder.py (hunk 1 above)
Comment (markddavidoff) [reply slack_pull_reminder.py:40]: It's moved to the generator at the top

File: slack_pull_reminder.py (hunk 2 above)
Comment (markddavidoff) [reply slack_pull_reminder.py]: done!

File: slack_pull_reminder.py (hunk 3 above)
Comment (markddavidoff) [reply slack_pull_reminder.py]: done!

**Comment by markddavidoff** [comment]
Added another pr for usernames as well. Thanks for this repo btw, saved me a bunch of time and effort. I've got this up and running on lambdas limited by repos and usernames for each of my org's teams (we don't use github teams, but that might be fun to add too)

**Comment by ekmartin** [comment]
Thanks!
